scripts/linters/html_linter.py

```diff
@@ -29,6 +29,8 @@
 from . import linter_utils
 from .. import common
 
+from scripts.linters.warranted_safe_pipe_usages import WARRANTED_USAGES  # isort:skip pylint: disable=import-only-modules
```

Comment:
Change it to `from . import warranted_safe_pipe_usages` and use `warranted_safe_pipe_usage.WARRANTED_USAGES` everywhere  instead of just `WARRANTED_USAGES`